skosprovider/skos.py

```diff
@@ -478,7 +478,7 @@ def __repr__(self):
         return "Collection('%s')" % self.id
 
 
-def label(labels=[], language='any', sortLabel=False):
+def label(labels=[], languages='any', sortLabel=False):
```

Comment:
I prefer leaving the parameter name as "language", otherwise it's a backwards incompatible fix. That's something for a version 2.0.0. This should just be a minor update to a version 1.1.0. The vast majority of usecases will be passing a single language.